docs/site/Building-frontend-angular-application.md

```diff
@@ -0,0 +1,146 @@
+---
+lang: en
+title: 'Building an Angular Application from OpenAPI Specification'
+keywords:
+  LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Angular, Frontend
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Building-frontend-angular-application.html
+---
+
+After a LoopBack application with REST APIs is created, the OpenAPI
+specification can be used to generate the models and web client in an Angular
+application using [ng-openapi-gen](https://www.npmjs.com/package/ng-openapi-gen)
+Node.js module.
```

Comment:
It's probably better to set some context before jumping into `ng-openapi-gen`? Such as:

```
If you create REST APIs using LoopBack and need to build a Front-end application that consumes the APIs, ...
```